rdkit/Chem/Draw/IPythonConsoleIntegration.py

```diff
@@ -0,0 +1,602 @@
+        for mol in self.selectedMolecules:
+            for confId in self.selectedConfIds:
+                yield Chem.MolToMolBlock(mol, confId=confId)
+                
+    @property
+    def allConfIds(self):
+        """ Return the number of conformations - use the first selected molecule to determine """
+        nconfIds = self.selectedMolecules[0].GetNumConformers()
+        return list(range(nconfIds))
+    @property
+    def getPropPrecalculated(self):
+        """ Return the precalculated properties """
+        if len(self.selectedMolNames) == 1:
+            mol = self.moldict[list(self.selectedMolNames)[0]]
+            if len(mol.GetPropNames()) == 0:
+                return 'Not found'
+            else:
+                prop = [(prop,mol.GetProp(prop)) for prop in mol.GetPropNames()]
+                return dict(prop)
+        else:
+            return None
+    @property
+    def getPropRDKit(self):
+        """ Return the RDKit calculated properties """
+        if len(self.selectedMolNames) == 1:
+            mol = self.moldict[list(self.selectedMolNames)[0]]
+            prop = list()
+            for prop_name in PROP_RDKIT:
+                calculator = Descriptors.__dict__[prop_name]
+                prop.append((prop_name,calculator(mol)))
+            return dict(prop)
+        else:
+            return None
+    
+    
+def handleMolAndConf(molViewState):
+    """ Handles molecules and conformers in update3D function """
+    uid = molViewState.uid
+    molViewState.selectMolecules(globals()['selectAllMols_'+uid].value, 
+                                 globals()['selectMultiMols_'+uid].value,
+                                 globals()['molId_'+uid].value)
+    sMolNames = molViewState.selectedMolNames
+    globals()['selected_mols_view_'+uid].value = ', '.join(sMolNames)
+    molViewState.selectConformations(globals()['selectAllConfs_'+uid].value, 
+                                     globals()['selectMultiConfs_'+uid].value,
+                                     globals()['confId_'+uid].value)
+    globals()['confId_'+uid].options = molViewState.allConfIds
+    sConfIds = molViewState.selectedConfIds
+    globals()['selected_confs_view_'+uid].value = ', '.join([str(x) for x in sConfIds])
+    return molViewState
+    
+def handleProperty(molViewState):
+    """ Handles property in update3D function """
+    # For retrieving precalculated property
+    uid = molViewState.uid
+    preCalcProp = molViewState.getPropPrecalculated
+    if isinstance(preCalcProp, dict):
+        try:
+            globals()['prop_precalc_wg_'+uid].options = preCalcProp.keys()
+            prop=globals()['prop_precalc_wg_'+uid].value
+            globals()['prop_precalc_view_'+uid].value = prop + ' : ' + str(preCalcProp[prop])
+        except:
+            pass
+    elif preCalcProp == 'Not found':
+        try:
+            globals()['prop_precalc_view_'+uid].value = 'No precalculated property found!'
+        except:
+            pass
```

Comment:
Why do you use try/except here? 

Instead of having four separate blocks here, you can use one around the `if/elif/elif/else`  block.